clouds/snowflake/modules/test/h3/H3_POLYFILL_TABLE.test.js

```diff
@@ -0,0 +1,24 @@
+const { runQuery, deleteTable } = require('../../../common/test-utils');
+
+test('H3_POLYFILL_TABLE should work', async () => {
```

Comment:
I think this test can be improved in one of two ways:

## Option 1: Quick
Just change the name from `H3_POLYFILL_TABLE should work` to `H3_POLYFILL_TABLE should create a new table which stores the result of H3_POLYFILL`

## Option2:  In-depth
1. The name `H3_POLYFILL_TABLE should work` to `H3_POLYFILL_TABLE should create a new table with the output of H3_POLYFILL`

2. Use describe.each or test.each to pass in each mode (intersects, contains, center). Call the H3_POLYFILL_TABLE procedure using the new 'mode' parameter. 
3. Instead of testing against an expected row length, compare it to the output of an inline query to H3_POLYFILL (again, also using the mode parameter).